armada/conf/default.py

```diff
@@ -17,6 +17,17 @@
 from oslo_config import cfg
 
 default_options = [
+
+    cfg.ListOpt(
+        'armada_apply_roles',
+        default=['admin'],
+        help='IDs of approved API access roles.'),
+
+    cfg.StrOpt(
```

Comment:
rather than manually creating the keystone options manually, can we leverage oslo.config and import these options, thus exposing every param and reducing ongoing tech-debt.